src/gt4sd/frameworks/molgx/README.md

```diff
@@ -0,0 +1,132 @@
+# IBM Molecule Generation Experience (Community Version)
```

Comment:
this readme should go in a docstring that is then parsed in the documentation page